d2-schemas/src/main/pegasus/com/linkedin/d2/D2RelativeStrategyProperties.pdl

```diff
@@ -48,6 +48,18 @@ record D2RelativeStrategyProperties {
    */
   lowErrorRate: optional double
 
+  /**
+   * If the latest server reported load score is above this specified factor of the cluster average,
+   * we will decrease the health score by downStep.
+   */
+  relativeLoadHighThresholdFactor: optional double
+
+  /**
+   * If the latest server reported load score is under this specified factor of the cluster average,
+   * we will increase the health score by upStep.
```

Comment:
I think we can have more discussion on up/downStep model vs. P2C. Sticky routing is currently best effort. It's not guaranteed to reach the same host when the pointsMap changes. I still think that 5s maybe too long for server reported load to be effective.